core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2094,6 +2094,14 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.WORKER)
           .build();
+  public static final PropertyKey WORKER_NETWORK_READER_BUFFER_SIZE_BYTES =
```

Comment:
Should we call this max bytes in flight instead?

Reply:
This name comes from a property we had before moving to gRPC. I also plan to reuse it once the gRPC buffer size become configurable, at which time I will remove this workaround in favor of the built-in flow control mechanism(the RPC protocol will still be backward compatible).

Reply:
So this is a configuration option that was available in 1.x which did the same thing?

Reply:
The option in 1.x is `WORKER_NETWORK_READER_BUFFER_SIZE_PACKETS`. The flow control logic is similar except that it used receipt callback from Netty.